zh/backup-restore-overview.md

```diff
@@ -43,6 +43,7 @@ TiDB Operator 1.1 及以上版本推荐使用基于 CustomResourceDefinition (CR
 ### 通用字段介绍
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.toolImage`: 用于指定 `Backup` 使用的工具镜像, 例如使用 BR 备份时可以设置为 `spec.toolImage: pingcap/br:v4.0.10` 或者在使用 Dumpling 备份时设置为 `spec.toolImage: pingcap/dumpling:v4.0.10`. (在 v1.1.9 以后版本支持这项配置)
```

Comment:
```suggestion
* `.spec.toolImage`：用于指定 `Backup` 使用的工具镜像，例如使用 BR 备份时可以设置为 `spec.toolImage: pingcap/br:v4.0.10` 或者在使用 Dumpling 备份时设置为 `spec.toolImage: pingcap/dumpling:v4.0.10`。如果不指定，使用 BR 备份时默认使用 `pingcap/br:${tikv_version}` 进行备份，使用 Dumpling 备份时默认使用 https://github.com/pingcap/tidb-operator/blob/${operator_version}/images/tidb-backup-manager/Dockerfile 文件中 `TOOLKIT_V40` 指定的 Dumpling 版本进行备份。v1.1.9 及后续版本支持这项配置。
```

Reply:
Please also update the description for `Restore` CR.

Reply:
ok